binding/python/test_porcupine.py

```diff
@@ -73,7 +45,37 @@ def test_process_multiple(self):
 
         porcupine.delete()
 
-        self.assertEqual(results, [6, 0, 1, 2, 3, 4, 5, 6, 7])
+        self.assertEqual(results, ground_truth)   
+
+    def test_en_0(self):
```

Comment:
be descriptive as much as possible for naming variables/functions:

```suggestion
    def test_en_single_keyword(self):
```